### DISCUSSION TITLE
rendering

### DISCUSSION THREAD

**Discussion by LorDCristhian** [Syslifters/sysreptor#310] [General]
Good afternoon community.

I just updated sysreptor from version 2023 to 2024 and now it no longer shows the "report.title" object reference when rendering the pdf. 

When I enter it outside the table, the reference appears, does anyone have a solution to this?

here my code:

`<table class="default">
  <tr>
    <th>Tipo de Prueba de Penetracion</th>
    <td>Caja negra</th>
  </tr>
  <tr>
    <th>Nivel de Prueba</th>
    <td>Parcial o General</td>
  </tr>
  <tr>
    <th>Activo</th>
    <td>{{ report.title }} </td>
  </tr>
  <tr>
    <th>Direccion IP</th>
    <td>127.0.0.1</td>
  </tr>
  <tr>
    <th>Retest</th>
    <td v-if="report.is_retest == true">Si</td>
    <td v-if="report.is_retest == false">No</td>
  </tr>
</table>`

img.
![reender](https://github.com/user-attachments/assets/fdacc961-6330-46cf-8e55-66a21eb1b106)

**Comment by MWedl** [comment]
Hi,
some versions ago we changed the parsing of curly braces and Vue template variables in markdown (see #228). Because of that, template variables inside multiline HTML blocks in markdown texts are no longer detected as such. Unfortunately, this is a limitation of our markdown parser because it cannot parse markdown syntax expressions (such as template variables like `{{ report.title }}`) inside HTML. 

In order to use template variable expressions inside HTML blocks in markdown, you need to break out HTML block by separating HTML and markdown blocks via two newlines. The template variable expression is then parsed as markdown and can be resolved correctly during PDF rendering.

Adapt your table code as follows:
```html
...
<tr>
    <th>Activo</th>
    <td>
        
        {{ report.title }}

    </td>
 </tr>
...
```
